### DISCUSSION TITLE
Does PdfMe have a size limit of file ?

### DISCUSSION THREAD

**Discussion by SivaNithish** [pdfme/pdfme#87] [Q&A]
Hello,

i tried the playground and played with it, to understand how things works. And it's pretty cool !

But i got an issue when i tried to upload some basePdf. I tried a pdf of 50 pages with a size of 1Mb and then i tried with a file of 122 pages and a size of 8 Mb. 

And the app has just crashed and a modal box of the browser says if we want to interrupt or no the process. 

Is this already happend to someone ? If yes, is it a settings to change or the app can't use a pdf of more than 1 Mb ~

**Comment by hand-dot** [comment]
Hi, @SivaNithish  Thank you for using pdfme.

There is no specific file size limit; it depends on the machine power you are running.
